packages/paste-icons/src/helpers/IconWrapper.tsx

```diff
@@ -7,15 +7,8 @@ export interface IconWrapperProps extends BoxElementProps, Pick<BoxStyleProps, '
   size?: IconSize;
 }
 
-const IconWrapper: React.FC<IconWrapperProps> = ({
-  as = 'span',
-  color = 'currentColor',
-  display = 'block',
-  element = 'ICON',
-  size = 'sizeIcon30',
-  ...props
-}) => {
-  return (
+const IconWrapper = React.forwardRef<HTMLElement, IconWrapperProps>(
```

Comment:
issue: we're missing the named function here `function IconWrapper({as ...`

Reply:
I dont think we actually need to do this. Looking at the avatar storybook - IconWrapper's display name shows up as expected without naming the function.